### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

**Comment by janisozaur** [comment]
I will clean it up a bit

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  createrelease:
+    name: createrelease
```
Comment (janisozaur) [inline .github/workflows/ci.yml:5]: might need to be limited to push w/tag only

**Review by tupaschoal — COMMENTED** [review]
Is this a conversion from the appveyor stuff (to retire the latter)?

**Comment by janisozaur** [comment]
Yes. Awaiting the JSON pr to be merged first before continuing here

**Comment by duncanspumpkin** [comment]
I want to apply something similar to the replay repo. Could you point me to/explain how this works and when it runs

**Pull request closed without merging**